README.md

```diff
@@ -0,0 +1,17 @@
+# nex2geotiff-injest
+
+Moves NETCDF data in s3 to another s3 bucket.
```

Comment:
Let's expand on this a bit, since this is too generic of a title.

What data does it pull down?
What data does it move?
What transformation does it apply? e.g. nex to geotiff, and how it slices the nex data into multiple geotiffs